### ISSUE TITLE
[Docs - New Feature]: shallow cloning

### ISSUE THREAD

**Issue by akosyakov** [gitpod-io/website#1278]
### Describe your feature

We should clarify that we do shallow clone during workspace startup and unshallow in the background in regular runnig workspace.

### How to test

See https://github.com/gitpod-io/gitpod/issues/6796

**Comment by fmartins** [comment]
Have you considered adding a system property that allows disabling the unshallow background task?

**Comment by Siddhant-K-code** [comment]
@loujaybee Any Plans for this?

**Comment by loujaybee** [comment]
@Siddhant-K-code -> Are there specific user or customer demands for this issue for bumping? Just trying to establish some more context and the use cases.

**Comment by Siddhant-K-code** [comment]
> Are there specific user or customer demands for this issue for bumping

No. Was cleaning around our Website Repo. to classify Product Documentation work. And, looking at the detailed discussion on https://github.com/gitpod-io/gitpod/issues/6796 that needs to be documented, but please take it as a low priority 🙏🏽
